platform/admin/ui/src/main/webapp/js/controllers/Modal.controller.js

```diff
@@ -13,19 +13,26 @@
 define(['jquery',
         'underscore',
         'marionette',
-        'js/wreqr'
-    ], function ($, _, Marionette, wreqr) {
+        'js/wreqr',
+        'js/views/SessionTimeoutModal.view'
+    ], function ($, _, Marionette, wreqr, SessionTimeoutView) {
         'use strict';
         var ModalController;
 
         ModalController = Marionette.Controller.extend({
             initialize: function (options) {
                 this.application = options.application;
                 this.listenTo(wreqr.vent, "showModal", this.showModal);
+                this.listenTo(wreqr.vent, "showSessionTimeoutModal", this.showSessionTimeoutModal);
```

Comment:
Similar to how we eliminated the `wreqr` in Intrigue, we should do the same here. 

Require in the sessionTimeout model since it's a singleton and listen to the change on `showPrompt`.  If it's true, show the modal, if it's not hide it.